invenio_app_ils/ui/src/invenio_app_ils/common/config/invenioConfig.js

```diff
@@ -1,8 +1,8 @@
 export const invenioConfig = {
   circulation: {
     deliveryMethods: {
-      DELIVERY: 'Have it delivered to my office',
       PICKUP: 'Pick it up at the library desk',
+      DELIVERY: 'Have it delivered to my office',
```

Comment:
we should for a list with a `default` instead, but we can address all these in one shot.